src/modules/RootPart.rb

```diff
@@ -1805,8 +1805,8 @@ def MountPartitions(root_device_current)
         # in the target map of the storage module
         AddToTargetMap()
         if Yast2::FsSnapshot.configured?
-          snapshot = Yast2::FsSnapshot.create_pre("before upgrade")
-          Yast2::FsSnapshotStore.save("upgrade", snapshot.number)
+          snapshot = Yast2::FsSnapshot.create_pre("before update", cleanup: :number, important: true)
+          Yast2::FsSnapshotStore.save("update", snapshot.number)
```

Comment:
why update instead of upgrade? I thought that SLE11->SLE12 or SLE12->SLE13 or SP1->SP2 is upgrade and not update.